src/main/java/frc/robot/Robot.java

```diff
@@ -6,139 +6,137 @@
 /*----------------------------------------------------------------------------*/
 
 package frc.robot;
+
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj.GenericHID.RumbleType;
 import edu.wpi.first.cameraserver.CameraServer;
-//import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.Timer;
 
 /**
- * The VM is configured to automatically run this class, and to call the
- * functions corresponding to each mode, as described in the TimedRobot
- * documentation. If you change the name of this class or the package after
- * creating this project, you must also update the manifest file in the resource
- * directory.
- */
+* The VM is configured to automatically run this class, and to call the
+* functions corresponding to each mode, as described in the TimedRobot
+* documentation. If you change the name of this class or the package after
+* creating this project, you must also update the manifest file in the resource
+* directory.
+*/
 public class Robot extends TimedRobot {
   private final Joystick joystick = new Joystick(0);
-  //private double kP = 0.5;  // This was for a pid that is currently not finished...
-  //private double kI = 0.5;
-  //private double kD = 0.5;
-  //private int setpoint = 0;
-  
-  /**
-   * This function is run when the robot is first started up and should be
-   * used for any initialization code.
-   */
+  private final Timer m_timer = new Timer();
+  private int SecondsLeft = 150;
```

Comment:
Since this is a constant, how about making it a private static final int? Also the convention for constants in Java is to use capitalization, e.g. GAME_IN_SECONDS?